db/blob/blob_fetcher.h

```diff
@@ -32,6 +33,15 @@ class BlobFetcher {
 
  private:
   const Version* version_;
-  ReadOptions read_options_;
+  const ReadOptions& read_options_;
 };
+
+class BlobFetcherCopyReadOptions : public BlobFetcher {
+  const ReadOptions read_options_copy_;
```

Comment:
nit suggestion: how about moving the data members to be after the function members and set its private section explicitly?

Reply:
It's OK, done!

Reply:
> nit suggestion: how about moving the data members to be after the function members and set its private section explicitly?

Done!